Do not create the shm node since Marcus' SHM implementation uses a directory
there.
---
 sutils/MAKEDEV.sh |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/sutils/MAKEDEV.sh
+++ b/sutils/MAKEDEV.sh
@@ -100,7 +100,7 @@ function mkdev {
 	;;
 
       std)
-	mkdev console tty urandom null zero full fd time mem klog shm
+	mkdev console tty urandom null zero full fd time mem klog
 	;;
       console|com[0-9])
 	st $I root 600 /hurd/term ${DEVDIR}/$I device $I;;
